pkgs/development/libraries/mesa/default.nix

```diff
@@ -65,6 +65,10 @@ stdenv.mkDerivation {
       url = "https://gitlab.freedesktop.org/mesa/mesa/commit/aebbf819df6d1e.patch";
       sha256 = "17248hyzg43d73c86p077m4lv1pkncaycr3l27hwv9k4ija9zl8q";
     })
+  ] ++ optionals (stdenv.isDarwin && stdenv.isAarch64) [
+    # Fix aarch64-darwin build, remove when upstreaam supports it out of the box.
+    # See: https://gitlab.freedesktop.org/mesa/mesa/-/issues/1020
+    ./aarch64-darwin.patch
```

Comment:
Can this be applied on all platforms? If you please remove the conditional.

Reply:
See https://github.com/NixOS/nixpkgs/pull/113754#issuecomment-782726994. If it works on `x86_64-darwin` I could also make it unconditional during the next regular update.